force-app/main/default/classes/TaxCalculation.cls

```diff
@@ -0,0 +1,58 @@
+public with sharing class TaxCalculation {
+    @AuraEnabled(cacheable=true)
+    public static Decimal calculateFederalTax(Decimal salary) {
+        Decimal federalTax = 0;
+
+        Decimal salaryMinusStandardDeduction = salary - 13850;
+
+        if (salaryMinusStandardDeduction == null || salaryMinusStandardDeduction <= 0) {
+            return federalTax;
+        } else if (salaryMinusStandardDeduction <= 11000) {
```

Comment:
As discussed in the presentation, custom metadata would be a nice enhancement to store tax data that might change year over year